From 18ab23cad1c88704e8be1cd5ef249c6c0af3d612 Mon Sep 17 00:00:00 2001
From: Emmanuele Bassi <ebassi@gnome.org>
Date: Tue, 5 Feb 2019 15:18:41 +0100
Subject: [PATCH 5/9] menu: Fix condition check

A logical "and" became a binary "and" in commit c35878ec.

(cherry picked from commit d9d3ec7cad305d169b1f5df3c3e9e0fd5fbf6922)
---
 gtk/gtkmenu.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gtk/gtkmenu.c b/gtk/gtkmenu.c
index f079f06e91..7b156f9861 100644
--- a/gtk/gtkmenu.c
+++ b/gtk/gtkmenu.c
@@ -5392,7 +5392,7 @@ gtk_menu_scroll_to (GtkMenu           *menu,
           if (!priv->upper_arrow_visible || !priv->lower_arrow_visible)
             gtk_widget_queue_draw (GTK_WIDGET (menu));
 
-          if (!priv->upper_arrow_visible &
+          if (!priv->upper_arrow_visible &&
               flags & GTK_MENU_SCROLL_FLAG_ADAPT)
             should_offset_by_arrow = TRUE;
           else
-- 
2.17.0

